packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx

```diff
@@ -0,0 +1,627 @@
+                    {
+                      ariaLabel: `${this._segments[arc.segmentIndex].legend}, ${
+                        this._segments[arc.segmentIndex].size
+                      } out of ${maxValue - this._minValue} or ${(
+                        (this._segments[arc.segmentIndex].size / (maxValue - this._minValue)) *
+                        100
+                      ).toFixed()}%`,
+                      ...this._segments[arc.segmentIndex].accessibilityData,
+                    },
+                    'img',
+                    true,
+                  )}
+                  onFocus={e => this._handleFocus(e, this._segments[arc.segmentIndex].legend)}
+                  onBlur={this._handleBlur}
+                  onMouseEnter={e => this._handleMouseOver(e, this._segments[arc.segmentIndex].legend)}
+                  onMouseMove={e => this._handleMouseOver(e, this._segments[arc.segmentIndex].legend)}
+                />
+              ))}
+              {this._renderNeedle()}
+              <g
+                onMouseEnter={e => this._handleMouseOver(e, 'Chart value')}
+                onMouseMove={e => this._handleMouseOver(e, 'Chart value')}
+              >
+                <TooltipText
+                  content={
+                    this.props.chartValueFormat === GaugeValueFormat.Fraction
+                      ? `${this._sweepFraction[0]}/${this._sweepFraction[1]}`
+                      : `${((this._sweepFraction[0] / this._sweepFraction[1]) * 100).toFixed()}%`
+                  }
+                  textProps={{
+                    x: 0,
+                    y: 0,
+                    textAnchor: 'middle',
+                    className: this._classNames.chartValue,
+                    role: 'img',
+                    'aria-label': `Current value: ${this._sweepFraction[0]} out of ${this._sweepFraction[1]}, or ${(
+                      (this._sweepFraction[0] / this._sweepFraction[1]) *
+                      100
+                    ).toFixed()}%`,
+                  }}
+                  maxWidth={this._innerRadius * 2 - 24}
+                  wrapContent={this._wrapContent}
+                />
+              </g>
+              {this.props.sublabel && (
+                <TooltipText
+                  content={this.props.sublabel}
+                  textProps={{
+                    x: 0,
+                    y: 4,
+                    textAnchor: 'middle',
+                    dominantBaseline: 'hanging',
+                    className: this._classNames.sublabel,
+                    role: 'img',
+                    'aria-label': this.props.sublabel,
+                  }}
+                  maxWidth={this._innerRadius * 2}
+                  wrapContent={this._wrapContent}
+                />
+              )}
+            </g>
+          </svg>
+        </FocusZone>
+        {this._renderLegends()}
+        {!this.props.hideTooltip && this.state.isCalloutVisible && (
+          <Callout
```

Comment:
In other charts we don't show the hover card if the legend for a segment is un selected. Here we are showing the callout for all segments regardless of it being selected or not.

Reply:
fixed